From bd9fcb4f4b5b74c38dc1e11a236cf4b4db58374e Mon Sep 17 00:00:00 2001
From: lifeich1 <lifeich0@gmail.com>
Date: Sun, 7 Dec 2025 08:20:37 +0800
Subject: [PATCH] for moran

---
 Makefile          | 4 ++++
 all-packages.conf | 6 +++---
 2 files changed, 7 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 6163fee..7346a11 100644
--- a/Makefile
+++ b/Makefile
@@ -48,6 +48,10 @@ install:
 	  install -d $(DESTDIR)$(RIME_DATA_DIR)/opencc; \
 	  install -m 644 $(OUTPUT)/opencc/*.* $(DESTDIR)$(RIME_DATA_DIR)/opencc; \
 	fi
+	@if [ -d "$(OUTPUT)/lua" ]; then \
+	  install -d $(DESTDIR)$(RIME_DATA_DIR)/lua; \
+	  install -m 644 $(OUTPUT)/lua/*.* $(DESTDIR)$(RIME_DATA_DIR)/lua; \
+	fi
 
 clean:
 	rm -rf $(OUTPUT) > /dev/null 2>&1 || true
diff --git a/all-packages.conf b/all-packages.conf
index 82387de..bca8691 100755
--- a/all-packages.conf
+++ b/all-packages.conf
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-package_list=()
+package_list=(moran)
 
-source "${root_dir:-.}"/preset-packages.conf
-source "${root_dir:-.}"/extra-packages.conf
+# source "${root_dir:-.}"/preset-packages.conf
+# source "${root_dir:-.}"/extra-packages.conf
-- 
2.51.2

